jib-cli/src/main/java/com/google/cloud/tools/jib/cli/buildfile/BuildFiles.java

```diff
@@ -41,6 +41,10 @@
 /** Class to convert BuildFiles to build container representations. */
 public class BuildFiles {
 
+  private BuildFiles() {
+    throw new IllegalStateException("Utility class, should not be instantiated.");
+  }
```

Comment:
Unless SonarCloud insists, I think we should just do
```
private BuildFiles() {}
```

Reply:
I think SonarCloud will be happy as long as we have a private constructor.  I just made the change.

* and my mistake, this is a "code smell" rather than a "bug" in sonar's dashboard.